### PR TITLE
Added meeting specific fields for users.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

**Comment by normanjaeckel** [comment]
Still open: initial data / example data have to be updated.

**Comment by normanjaeckel** [comment]
@emanuelschuetze and @FinnStutzenstein Please check again.

**Comment by normanjaeckel** [comment]
Ready to merge.

**Comment by normanjaeckel** [comment]
@tsiegleauq Please pay attention to this changes for initial data and client. ;-)

**Comment by ostcar** [comment]
Could you add a value to organisation_management_level for unprivileged user? For example:

```
  # Organisation, meeting and committee
  organisation_management_level:
    type: string
    description: Hierarchical permission level for the whole organisation.
    enum:
    - superuser
    - can_manage_organisation
    - can_manage_users
    - unprivileged
```

**Comment by FinnStutzenstein** [comment]
We decided to leave it empty if one does not have a level.

File: docs/models.yml [hunk 1]
```diff
@@ -103,28 +97,23 @@ user:
...
   password: string
   default_password: string
-  about_me: HTMLStrict
   gender: string
-  comment: HTMLStrict
-  number: string
-  structure_level: string
   email: string
+  default_number: string
+  default_structure_level: string
```
Comment (FinnStutzenstein) [inline docs/models.yml:103]: default_vote_weight is missing.

**Comment by FinnStutzenstein** [comment]
Since @tsiegleauq also kept the `docker/initial-data.json` up to date, please also do the changes there.

**Comment by ostcar** [comment]
Could you merge this? It would help me in the permission-service

**Comment by FinnStutzenstein** [comment]
> Could you merge this?

TODO:
- rebase
- adjusting the `initial-data.json`.

**Comment by normanjaeckel** [comment]
Ready to merge.

File: docs/models.yml (hunk 1 above)
Comment (normanjaeckel) [reply docs/models.yml:103]: Done.

**Pull request merged**